File: seeker/snippet/Menu.java
//date: 2023-05-30T16:40:28Z
//url: https://api.github.com/gists/1b4e340655fde4517ac769c270630146
//owner: https://api.github.com/users/asengsaragih

package com.suncode.relicbatik.model;

import android.graphics.drawable.Drawable;

public class Menu {
    private final String title;
    private final String description;
    private final Drawable icon;

    public Menu(String title, String description, Drawable icon) {
        this.title = title;
        this.description = description;
        this.icon = icon;
    }

    public String getTitle() {
        return title;
    }

    public String getDescription() {
        return description;
    }

    public Drawable getIcon() {
        return icon;
    }
}

File: seeker/snippet/menu.py
#date: 2023-08-04T16:56:08Z
#url: https://api.github.com/gists/d56e95176b697d5fdeb9a3ad6cd16266
#owner: https://api.github.com/users/AitsuYuyu

import workers

def menuPrincipal():
    while True:
        print("---------------------------")
        print("         MENU PRINCIPAL    ")
        print("---------------------------")
        
        print("1. Registrar empleado")
        print("2. Listar empleados")
        print("3. Modificar empleados")
        print("4. Despedir empleados")
        print("5. Registrar entrada o salida")
        print("0. Salir")
        
        try:
            option = int(input("Digite la opcion a realizar: "))
        except ValueError:
            print("Ingrese un valor numerico")
        else:
            if option == 1:
                workers.register()
            elif option == 2:
                workers.listar()
            elif option == 3:
                workers.modificarEmpleado()
            elif option == 4:
                workers.despedir_empleado()
            elif option == 5:
                workers.in_out()            
            elif option == 0:
                print("Saliendo!")
                break
            else:
                print("opcion invalida")